### DISCUSSION TITLE
Does WASP do a QC on SNPs in VCF files?

### DISCUSSION THREAD

**Discussion by shadrinams** [alexdobin/STAR#1851] [Q&A]
Hello,

I have looked through posts here and did not find information on if/how the WASP run in STAR does filtering of SNPs in a provided VCF.
As I understand, WASP uses only heterozygous SNPs (based on the GT field). Does it use other fields like FILTER=PASS, QUAL, GQ and so on?
If I want to do an additional quality check of heterozygous variants, should I filter my VCF file before the STAR step?

(I am planning to mark duplicates with PICARD next, and then to run GATK ASEReadCounter, following the procedure published here: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02122-z)

Thank you!

Best regards,
Maria.

**Comment by alexdobin** [comment]
Hi Maria,

except for heterozygous SNVs, no additional filtering is done. You need to supply VCF with only those variants you want to be considered.
